sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/WebJobsEventHubTestBase.cs

```diff
@@ -73,28 +73,22 @@ protected void ConfigureTestEventHub(IHostBuilder builder)
                         {"AzureWebJobsStorage", StorageTestEnvironment.Instance.StorageConnectionString}
                     });
                 })
-                .ConfigureServices(services =>
-                {
-                    services.Configure<EventHubOptions>(options =>
-                    {
-                        options.CheckpointContainer = Guid.NewGuid().ToString("D").Substring(0, 13);
-                        // Speedup shutdown
-                        options.EventProcessorOptions.MaximumWaitTime = TimeSpan.FromSeconds(5);
-                    });
-                })
                 .ConfigureDefaultTestHost<T>(b =>
                 {
                     b.AddEventHubs(options =>
                     {
                         options.EventProcessorOptions.TrackLastEnqueuedEventProperties = true;
+                        options.EventProcessorOptions.MaximumWaitTime = TimeSpan.FromSeconds(5);
+                        options.CheckpointContainer = Guid.NewGuid().ToString("D").Substring(0, 13);
                     });
                 })
                 .ConfigureLogging(b =>
                 {
                     b.SetMinimumLevel(LogLevel.Debug);
                 });
 
-            configurationDelegate(hostBuilder);
+            ConfigureTestEventHub(hostBuilder);
```

Comment:
Some tests rely on `ConfigureTestEventHub` not being executed when configuration delegate is provided.

Reply:
All tests are passing..

Reply:
The test in question are verifying the various connection settings. We didn't execute `ConfigureTestEventHub` because it adds the default connection settings so those test might keep passing even if the feature is borken.